tests/sentry/tasks/test_releasemonitor.py

```diff
@@ -313,6 +313,23 @@ def test_release_is_unadopted_with_sessions(self):
             unadopted__gte=now,
         ).exists()
 
+        # Make sure re-adopting works
+        self.bulk_store_sessions(
+            [
+                self.session_dict(i, self.project1, self.release.version, self.environment.name)
+                for i in range(50)
+            ]
+        )
+        time.sleep(1)
```

Comment:
Is this required for the tests to be stable?

Reply:
Yeah it wouldn't pass without them :( I tried making it lower too with no success.

Reply:
Hmm, is this caused by us needing to pass an older timestamp when creating sessions? I know that we need to do that when creating events so that results will be returned.